sandbox/src/mocks/r4/createServiceRequest/requests/createIncomplete.json

```diff
@@ -0,0 +1,87 @@
+{
+  "resourceType": "ServiceRequest",
+  "subject": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9912003888"
+    }
+  },
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/message-category-servicerequest",
+          "code": "referral"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/ServiceRequest-Code",
+        "code": "advice-and-guidance"
+      }
+    ]
+  },
+  "priority": "routine",
+  "intent": "order",
+  "status": "draft",
+  "extension": [
+    {
+      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-ERS-Referral",
+      "extension": [
+        {
+          "url": "sourceSystem",
```

Comment:
As per [here](https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?pageId=534350452), I would also question if this is needed.